<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HiddenBumpDialog</class>
 <widget class="QDialog" name="HiddenBumpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="l_wrongPsw">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>WRONG PASSWORD</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_wrongPsw_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PLEASE ENTER THE CORRECT  PASSWORD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_saveGeneralBumpTh">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>305</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="l_password">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="pB_login">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>322</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="l_generalBumpTh">
      <property name="text">
       <string>Hidden Bump Threshold:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lE_generalBumpTh"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pB_EditPassword">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
